src/main.py

```diff
@@ -24,6 +25,22 @@ def load_vcap():
     os.environ[uev_env_var] = uev_ups['credentials']['key']
 
 
+def decrypt_params(encrypted):
+    vcap_application = json.loads(os.getenv('VCAP_APPLICATION', '{}'))
+    vcap_services = json.loads(os.getenv('VCAP_SERVICES', '{}'))
+
+    space = vcap_application['space_name']
+
+    encryption_ups = next(
```

Comment:
Is there a realistic possibility that a matching service might not be found?

Reply:
Not really. We couple the service with the deployment and there would be a deploy error if the service didn't connect.